env2secret.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
I thought about dropping the Makefile and the need for this file as Helm is much better at handling release on k8s